pylintrc

```diff
@@ -413,16 +413,16 @@ max-spelling-suggestions=2
 max-args=10
 
 # Maximum number of locals for function / method body
```

Comment:
Could we use the default value and remove the description and the value ?

Reply:
You mean for all of them or for max locals in particular? For all of the settings in this PR at least, the configured value is higher than the default, so using the defaults would require a lot more disabling comments.

As for the rest of the config file, personally I would be happy to rip through and cut out a bunch of the text. Currently there's an awful lot of text, and I find it hard to read. How much cleanup were you thinking?

Reply:
I was thinking of removing the default values and their description. It should be possible to do that automatically with the conf génération command iirc

Reply:
I'm not quite sure what you mean.

Reply:
Okay, I cut a few more of them.